my-site.rb

```diff
@@ -3,6 +3,19 @@
 class MySite < Sinatra::Base
 
   get "/" do
+    erb :index
+  end
+
+  get "/projects.erb" do
```

Comment:
I might recommend that the routes are just the strings with the route name without the .erb extension